packages/x-data-grid-pro/src/tests/dataSourceLazyLoader.DataGridPro.test.tsx

```diff
@@ -171,13 +171,10 @@ describeSkipIf(isJSDOM)('<DataGridPro /> - Data source lazy loader', () => {
       // wait until the rows are rendered
       await waitFor(() => expect(getRow(0)).not.to.be.undefined);
 
-      await act(async () => {
-        apiRef.current?.scrollToIndexes({ rowIndex: 10 });
-      });
+      await act(async () => apiRef.current?.scrollToIndexes({ rowIndex: 10 }));
```

Comment:
Nitpick: It seems that the recommendation is to wrap the function return in a `{}`. 🤔 

```suggestion
      await act(async () => {
        apiRef.current?.scrollToIndexes({ rowIndex: 10 });
      });
```

Reply:
Where is the recommendation? 🤔 

In theory it shouldn't matter in this case.

Using `{}` requires async funcs to be awaited though. And our linting doesn't have that rule, so it could be more error-prone.

Returning the funcs directly should work for both promises and regular funcs, since the `await act` would await the entire chain.

Reply:
> Where is the recommendation? 🤔

https://react.dev/reference/react/act

I'm not suggesting that it might not work, just suggesting to have one approach, at least in a single file. 🙈